docs/maintainers/determining-concepts.md

```diff
@@ -23,11 +23,30 @@ Every new Concept introduced, regardless of how _trivial_ it may seem, must be e
 
 All Concepts covered in a Concept Exercise must either be taught by it, or must be marked as a prerequisite in the `config.json`.
 
+### Naming Concept Exercises
+
+Concept Exercises should be named after their story/theme, _not_ after its concept(s). The following rules apply to naming Concept Exercises:
+
+1. Use lowercase.
+1. Use [kebab-case][kebab-case].
+1. Use the original name when forking an exercise without major changes.
+
+Good examples:
+
+- `tim-from-marketing`
+- `lucians-luscious-lasagna`
+- `calculator-conundrum`
+
+Bad examples:
+
+- `TIM-FROM-MARKETING`: should use lowercase (i.e. `classes`)
+- `TimFromMarketing`: should use use kebab-case (i.e. `anonymous-functions`)
+
```

Comment:
You might consider adding a bad example that is named after the concept (as opposed to the story/theme).

Reply:
Ooh, excellent idea!